src/app-layout/drawer/index.tsx

```diff
@@ -119,9 +119,12 @@ export const Drawer = React.forwardRef(
         >
           {!isMobile && !hideOpenButton && regularOpenButton}
           <TagName
-            className={clsx(resizeHandle && styles['drawer-resize-content'])}
+            className={clsx(resizeHandle && styles['drawer-resize-content'], styles['drawer-content-wrapper'])}
             aria-label={mainLabel}
             aria-hidden={!isOpen}
+            style={{
+              ['blockSize']: `calc(100vh - ${(topOffset || 0) + (bottomOffset || 0)}px)`,
```

Comment:
Same, but cleaner/shorter

```suggestion
              blockSize: `calc(100vh - ${(topOffset || 0) + (bottomOffset || 0)}px)`,
```